lib/rubocop/cop/style/class_vars.rb

```diff
@@ -6,6 +6,29 @@ module Style
       # This cop checks for uses of class variables. Offenses
       # are signaled only on assignment to class variables to
       # reduce the number of offenses that would be reported.
+      #
+      # Setting value for class variable need to take care.
+      # If some class has been inherited by other classes, setting value
+      # for class variable affected children classes.
+      # So using class instance variable is better in almost case.
+      #
+      # @example
+      #   # bad
+      #   class A
+      #     @@test = 10
+      #   end
+      #
+      #   # good
+      #   class A
+      #     @test = 10
+      #   end
+      #
+      #   class A
+      #     def test
+      #       @@test # you can access class variable without offence
```

Comment:
This example gives an impression that by using @@test here you can access class-level instance vabiable set on line 23, which is not the case. Also, to access class variable without offence here you need to set it first, no? And that would be an offence to rubycop.